migration/export_database.py

```diff
@@ -29,13 +27,13 @@ def export_db():
     repeated_notices = []
     for notice in notices_cursor:
         try:
-            mc_new.get_default_database().notices.insert(notice)
-            print("inserted notice: ", notice)
+            mc_new.get_default_database().notices.insert_one(notice)
+            print(("inserted notice: ", notice))
```

Comment:
Is this bracket required?

Reply:
Seems unnecessary and it may change the output.
![image](https://github.com/metakgp/mftp/assets/34770591/3630b792-d10b-4fc2-a123-85703c38ffd5)